controller/Dockerfile

```diff
@@ -1,4 +1,6 @@
 FROM alpine:latest
 
 ADD controller /controller
+# The file is owned by root. Make sure non-root user can execute it
```

Comment:
```suggestion
# When running as non root and building in an environment that `umask` masks out '+x' for others, it won't be possible to execute. Make sure all can execute, just in case
```

Reply:
Amended this myself :)